CONTRIBUTING.md

```diff
@@ -65,41 +65,59 @@ Steps to get started:
 
 ## Getting your development environment set up
 
-After cloning the project there are a few steps required to get the project running.
+After cloning the project there are a few steps required to get the project running. 
+For running all tests, take the following steps:
 
-1.  Fetch submodules to pull ECMAScript test suites.
+1. Fetch submodules to pull ECMAScript test suites.
 
     ```bash
     git submodule update --init --recursive
     ```
 
-2.  Install js dependencies.
+2. Install js dependencies.
 
     ```bash
     yarn
     ( cd crates/swc_ecma_transforms; yarn )
     ```
 
-3.  Setup some environment variables which is required for tests.
+3. Setup some environment variables which is required for tests.
 
     ```bash
-    export RUST_BACKTRACE=full
+    export RUST_BACKTRACE=0
     export PATH="$PATH:$PWD/crates/swc_ecma_transforms/node_modules/.bin"
     export RUST_MIN_STACK=16777216
     ```
 
-4.  Install deno, if you are going to work on the bundler.
+4. Install deno
```

Comment:
This, and `wasm32-wasi` is required for running all tests successfully

Reply:
But if you are not going to work on the bundler, you don't need to run bundler tests.

Reply:
As someone pulling the code for the first time, I usually want to run all tests to make sure everything went smoothly.

I tried to cover this by calling out "to run all tests..." in this subsection header.

Maybe a better way is to have a different subsection that outlines required dependencies per package, steps for running all tests, and a callout for how to run individual package tests?

Reply:
@kdy1 thoughts?

Reply:
Oh, yeah you are right.